tests/test_utils.py

```diff
@@ -170,22 +173,61 @@ def test_export_json(self, tmpdir):
         assert expected == json_str
 
 
-    def test_load_schema(self, tmpdir, mocker):
+    def test_load_default(self, mocker):
+        
+        biothings_schema_path = "tests/data/etc/data_models/biothings.model.jsonld"
```

Comment:
I'm afraid that I wasn't clear in my previous review. My apologies. 

I wasn't thinking of duplicating the JSON-LD files in the `tests/` folder. You should still be able to use the `io_utils.load_default()` and `io_utils.load_schemaorg()` function directly, which should load the JSON-LD files from the `schematic/etc/` folder. You can then perform the same checks as you have here. 

Does that make sense? I just don't think we need to duplicate files in this case.

Reply:
Right, true, I was going back and forth on whether or not to duplicate the `etc/data_models` folder. I thought I should consolidate all "resources" used by pytest tests within the `tests/data` folder. But you're right, `etc` folder contains package-specific files and will always be present in the core without being modified, so we can use it directly.

Thanks for the review @BrunoGrandePhD, I've fixed it.